src/test/java/org/dataone/hashstore/filehashstore/FileHashStoreProtectedTest.java

```diff
@@ -643,8 +643,8 @@ public void validateTmpObject_validationRequested_matchingChecksum() throws Exce
     }
 
     /**
-     * Confirm validateTmpObject does not throw exception when requested to validate checksums with
-     * good values, and that the tmpFile passed is deleted.
+     * Confirm validateTmpObject throws exception when requested to validate a bad checksum,
```

Comment:
I am confused there. There is  no code change here, but you comment changed from "does not throw" to "throw", and "good values" to "bad values".

Reply:
Sorry for the confusion - the description was actually wrong (the junit test is correct), so I updated the javadoc.